dashboard/src/containers/RoutesContainer/Routes.tsx

```diff
@@ -63,20 +65,17 @@ class Routes extends React.Component<IRoutesProps> {
         {Object.entries(privateRoutes).map(([route, component]) => (
           <PrivateRouteContainer key={route} exact={true} path={route} component={component} />
         ))}
-        <PrivateRouteContainer
-          key={reposPath}
-          exact={true}
-          path={reposPath}
-          component={RepoListContainer}
-        />
```

Comment:
Was this a temporary route we'd added during the switch to clustered/namespaced routes? Right, then we've updated `reposPath` but never removed the now redundant route?

Reply:
seems so, there was no reason to keep it as a separated case.